test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json

```diff
@@ -36,5 +38,8 @@
   "compilationOptions": {
     "allowUnsafe": true
   },
-  "testRunner": "xunit"
+  "testRunner": "xunit",
+  "content": [
+    "TestResources/testCert.pfx"
```

Comment:
I copied the existing certificate file from Microsoft.AspNetCore.Server.KestrelTests.  Our tests already disable certificate validation since it's self-signed, and I suspect this would disable expiration validation as well.